tools/rdm/TestMixins.py

```diff
@@ -797,22 +853,22 @@ def ResetState(self):
     self._wrapper.Run()
 
 class GetZeroByteMixin(object):
-  """Get a single byte parameter with value 0, expect NR_DATA_OUT_OF_RANGE"""
+  """Get a parameter with value 0, defaults to a byte, expect NR_DATA_OUT_OF_RANGE"""
   CATEGORY = TestCategory.ERROR_CONDITIONS
+  DATA = struct.pack('!B', 0)
```

Comment:
Don't redefine this - Byte means 8 bits. If you want to create a GetZeroInt32Mixin I'm ok with that.